### PR TITLE
[Mm-27817] e2e add user to gm

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

e2e test for adding a user to an existing GM

#### Ticket Link

[MM-27817](https://mattermost.atlassian.net/browse/MM-27817)

#### aditional notes
since i renamed the file for consistency, let me know if you prefer to have a separate file, but those cases seemed related to me

### CODE REVIEW

File: e2e/cypress/integration/multi_team_and_dm/gm_add_user_spec.js [hunk 1]
```diff
@@ -70,6 +90,54 @@ describe('Multi-user group messages', () => {
...
+        cy.get('#selectItems input').type('{enter}');
+
+        // # Click Go
+        cy.get('button#saveItems').click({force: true});
+
+        // * modal closes
+        cy.get('#moreDmModal').should('not.be.visible');
+
+        // * original messages does not exist
+        cy.contains('.post-message__text', 'historical').should('not.be.visible');
```
Comment (sudheerDev) [inline e2e/cypress/integration/multi_team_and_dm/gm_add_user_spec.js:130]: One of the expected test steps is that it. should show. the message `This is the start of your group message history...".` after creating the group. Can you add it?

**Review by sudheerDev — CHANGES_REQUESTED** [review]

**Review by sudheerDev — APPROVED** [review]
LGTM

**Comment by Willyfrog** [comment]
sorry @prapti I forgot to push the last changes

**Review by prapti — APPROVED** [review]
Looks great! Thank you for adding the tests @Willyfrog!

**Pull request merged**